src/Initializer/InitProcedure/InitModel.cpp

```diff
@@ -244,6 +244,32 @@ static void initializeCellMatrices(LtsInfo& ltsInfo) {
 
   memoryManager.recordExecutionPaths(seissolParams.model.plasticity);
 #endif
+
+  //  const bool ITMToggle = memoryManager.getITMParameters()->getITMToggle();
+  auto itmParameters = seissol::SeisSol::main.getSeisSolParameters().itmParameters;
+
+  bool ITMToggle = itmParameters.ITMToggle;
```

Comment:
I think it is better to have it in `InitModel.cpp` rather than `InitSideConditions.cpp` as the side conditions are more related to the initial conditions IMO. ITM activation can be done theoretically on all kinds of initial conditions.